src/scenes/home/signup/signup.js

```diff
@@ -18,7 +18,7 @@ class SignUp extends Component {
     super(props);
     this.state = {
       email: '',
-      emailValid: true,
+      emailValid: false,
       error: false,
```

Comment:
Why is `emailValid` now set to false in the constructor, unlike the other validity states (`passwordValid`, `zipValid`)?  Your branch seems to work correctly when it's left true, how it originally was.

This is also inconsistent with the pattern used w/ the other validity states.

Reply:
I changed it to model the fact that the email will initially be empty and thus invalid. But yeah, it seems that it works correctly either way.

I think it works for either case because ```isValid``` is set to true in the constructor for ```FormInput``` and that takes preference over value of ```emailValid``` for display of invalid state. I tried changing ```isValid``` to false in the constructor but that just shows the invalid state before the user has a chance to type.

So, should I change ```emailValid``` back to true or change ```passwordValid``` and ```zipValid``` to false as well to show that they are empty and invalid initially?

Reply:
@babu-thomas I'm thinking that they should all be false, like as you say, they're empty and therefore invalid initially. As long as the invalid error message isn't shown before the user has a chance to type.